neo/Prompt/Commands/Wallet.py

```diff
@@ -305,6 +306,66 @@ def command_desc(self):
         return CommandDesc('unspent', 'show unspent assets', params=[p1, p2, p3, p4])
 
 
+class CommandWalletSplit(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        wallet = PromptData.Wallet
+
+        if len(arguments) < 4:
+            print("Please specify the required parameters")
+            return None
+
+        if len(arguments) > 5:
+            # the 5th argument is the optional attributes,
+            print("Too many parameters supplied. Please check your command")
+            return None
+
+        try:
+            from_addr = wallet.ToScriptHash(arguments[0])
+        except ValueError as e:
+            print(str(e))
```

Comment:
I think we might want to add a check with `isValidPublicAddress()` and print a user friendly message. Right now an invalid address gives a confusing error:
```
neo> wallet split bad_addr NEO n 2
subsection not found
```